pom.xml

```diff
@@ -483,8 +483,9 @@
     <version.org.codehaus.plexus.plexus-io>3.0.0</version.org.codehaus.plexus.plexus-io>
     <version.org.apache.maven.shared.maven-artifact-transfer>0.9.1
     </version.org.apache.maven.shared.maven-artifact-transfer>
-    <version.org.apache.maven.plugins.maven-compiler-plugin>3.7.0
+    <version.org.apache.maven.plugins.maven-compiler-plugin>3.8.1
     </version.org.apache.maven.plugins.maven-compiler-plugin>
+    <version.compiler.plugin>3.8.1</version.compiler.plugin>
```

Comment:
duplicate definition?

Reply:
We need this to override the jboss-parent definition:
https://github.com/jboss/jboss-parent-pom/blob/master/pom.xml#L316

Reply:
yes, but we have now 2 version properties in kie-parent, so I would recommend to remove one - `version.org.apache.maven.plugins.maven-compiler-plugin` as that is obviously not used or wrongly used ;)

Reply:
Updated to remove `version.org.apache.maven.plugins.maven-compiler-plugin`